tests/src/Functional/DefaultOgImageTest.php

```diff
@@ -0,0 +1,66 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Drupal\Tests\helfi_platform_config\Functional;
+
+use Drupal\Core\Language\LanguageInterface;
+use Drupal\language\Entity\ConfigurableLanguage;
+use Drupal\Tests\BrowserTestBase;
+
+/**
+ * Tests default og image.
+ */
+class DefaultOgImageTest extends BrowserTestBase {
+
+  /**
+   * {@inheritdoc}
+   */
+  protected $defaultTheme = 'stark';
+
+  /**
+   * {@inheritdoc}
+   */
+  protected static $modules = [
+    'language',
+    'helfi_platform_config',
+    'helfi_image_styles',
+  ];
+
+  /**
+   * {@inheritdoc}
+   */
+  public function setUp() : void {
+    parent::setUp();
+    foreach (['fi', 'sv'] as $langcode) {
+      ConfigurableLanguage::createFromLangcode($langcode)->save();
+    }
+  }
+
+  /**
+   * Tests [site:shareable-image] token.
+   */
+  public function testDefaultOgImage() : void {
+    /** @var \Drupal\Core\Utility\Token $token_service */
+    $token_service = $this->container->get('token');
+
+    /** @var \Drupal\language\ConfigurableLanguageManagerInterface $language_manager */
+    $language_manager = $this->container->get('language_manager');
+
+    foreach (['fi', 'sv'] as $langcode) {
+      // Set content language.
+      $language_manager->setCurrentLanguage(
+        $language_manager->getLanguage($langcode),
+        LanguageInterface::TYPE_CONTENT,
+      );
+
+      $og_image_url = $token_service->replace('[site:shareable-image]');
+
+      $this->drupalGet($og_image_url);
+
+      // Image should exist.
+      $this->assertSession()->statusCodeEquals(200);
```

Comment:
We should test the actual image style URL here to make sure the image is actually run through image style.